samples/json_sample.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package samples
+
+import (
+	"fmt"
+	"gorm.io/gorm"
+	"io"
+	"time"
+
+	"cloud.google.com/go/spanner"
+	samples "github.com/googleapis/go-gorm-spanner/samples/interleave"
+)
+
+type VenueDetails struct {
```

Comment:
Move this type definition to the `sample.go` file. That is where all the model definitions are, and this should go along with that.